.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.3.7
```

Comment:
@aclemons Remind me, why the downgrade to 2.3.7 again? (If anything, we should try to move ahead to 2.5.2 or 2.5.3 if we can)

Reply:
I was just targeting the oldest currently supported ruby version.

Reply:
Got it. I'm okay with this for the time being. 👍